packages/blockchain-wallet-v4-frontend/src/modals/BuySell/model.tsx

```diff
@@ -175,17 +175,7 @@ export const getPaymentMethod = ({
       )
     case BSPaymentTypes.FUNDS:
       if (orderType === 'BUY') {
-        const coinName = window.coins[counterCurrency]?.coinfig.name ?? counterCurrency
-
-        return (
-          <FormattedMessage
-            id='modals.simplebuy.confirm.funds_wallet'
-            defaultMessage='{coin} Wallet'
-            values={{
-              coin: coinName
-            }}
-          />
-        )
+        return window.coins[counterCurrency]?.coinfig.name ?? counterCurrency
```

Comment:
is `coinfig` ok or is it a typo?

Reply:
Unluckily? it is haha
<img width="184" alt="image" src="https://user-images.githubusercontent.com/97310750/179022177-9f94954b-4834-4f36-b611-995e84bad207.png">

Reply:
nope, is the attribute name